CRM/Dedupe/Merger.php

```diff
@@ -1164,15 +1108,11 @@ public static function getRowsElementsAndInfo($mainId, $otherId, $checkPermissio
             $label = ts('[x]');
           }
         }
-        elseif ($field == 'individual_prefix' || $field == 'prefix_id') {
+        elseif ($field == 'prefix_id') {
```

Comment:
@eileenmcnaughton  Do you know what it was checking for both of these before? I haven't seen a 'Bad Thing' emerge from this change though.